### PR TITLE
set default lang=en on login

### PR DESCRIPTION
**Pull request by vtni**
if your local chrome has set another default language, login does not work.

### CODE REVIEW

File: py3pin/Pinterest.py [hunk 1]
```diff
@@ -126,6 +126,7 @@ def login(self, headless=True, wait_time=15, proxy=None):
         chrome_options = Options()
         if headless:
             chrome_options.add_argument("--headless")
+            chrome_options.add_argument("--lang=en")
```
Comment (bstoilov) [inline py3pin/Pinterest.py:129]: Should this be outside of the if? If login is not headles

**Comment by bstoilov** [comment]
@vtni Thanks for spotting this. Is this only for headless login?

File: py3pin/Pinterest.py (hunk 1 above)
Comment (vtni) [reply py3pin/Pinterest.py:129]: I guess it would make sense in both cases, but yes

**Comment by imgVOID** [comment]
Thank you I exactly have not enlish chrome default language and login problems

File: py3pin/Pinterest.py (hunk 1 above)
Comment (imgVOID) [reply py3pin/Pinterest.py:129]: Both cases

**Review by imgVOID — COMMENTED** [review]
However my login problems has been caused not because there was another language settings.

**Comment by vtni** [comment]
I have added the language as a parameter to the function. That could be used, if you want to log in with "en-US"  or "en-GB" instead.

**Pull request merged**